es6-shim.js

```diff
@@ -348,6 +361,43 @@
       return it;
     },
 
+    GetMethod: function (o, p) {
+      var func = (ES.ToObject(o))[p];
```

Comment:
why the wrapping parens? `ES.ToObject(o)[p]`, or even better, an intermediate variable for `ES.ToObject(o)`?